schema-source/RS0001.schema.yaml

```diff
@@ -300,32 +300,32 @@ GridVariablesCooling:
       Data Type: "[Numeric][1..]"
       Units: "m3/s"
       Constraints: ">0.0"
-      Required: "if condenser_type = LIQUID"
```

Comment:
So far our conditional requirements have all relied on data elements in the same data group. If we want to refer to a data element in a parent (or other ancestor) data group, 232 will need to establish a syntax for doing so.

I've captured this in the Lattice repo: https://github.com/bigladder/lattice/issues/48